platform/broadcom/sonic-platform-modules-delta/agc032/scripts/agc032_platform_init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#platform init script for Delta agc032
+
+#fan speed monitol start
+/usr/share/sonic/device/x86_64-delta_agc032-r0/fancontrol.service /usr/share/sonic/device/x86_64-delta_agc032-r0/fancontrol &
+
+#Get syseeprom information
+echo 0 |sudo tee /sys/devices/platform/delta-agc032-cpupld.0/cpu_i2c_mux_sel
+decode-syseeprom
+echo 1 |sudo tee /sys/devices/platform/delta-agc032-cpupld.0/cpu_i2c_mux_sel
```

Comment:
Why do this at init?

Reply:
Because of in our hardware design, 
syseeprom is on the different mux with mainboard after CPLD,
we have to switch CPLD mux from main board to syseeprom before getting syseeprom data.

The purpose of switch CPLD mux and getting syseeprom information at init script is to save data in  syseeprom_cache. 
After that, we can read "syseeprom_cache" directly when executing command "show platform syseeprom", "sudo decode-syseeprom", "show version".

Reply:
I see. Can you update the comment to mention this, because if the caching behavior ever changes there could be an issue. It will be easier to locate with a comment.

Reply:
Sure, i have updated the comment, please check it again, thanks.